lld/test/COFF/Inputs/loadconfig-arm64ec.s

```diff
@@ -1,4 +1,4 @@
-        .section .rdata,"dr"
```

Comment:
Do we lose test coverage by removing `.rdata`? If not, LGTM

Reply:
I think we don't lose coverage. We still have other tests that declare _load_config_used in .rdata, so we should be protected from somehow accidentally depending on it being in .00cfg. I made the change to Inputs/ to widen coverage of merging and making shared load config more similar to real-world case feels right.

Thanks for reviews!